src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_validators.py

```diff
@@ -53,13 +99,25 @@ def validate_resource_types(string):
         raise ValueError
     return ResourceTypes(_str=''.join(set(string)))
 
+def validate_select(namespace):
+    if namespace.select:
+        namespace.select = ','.join(namespace.select)
+
 def validate_services(string):
     ''' Validates that services string contains only a combination
     of (b)lob, (q)ueue, (t)able, (f)ile '''
     if set(string) - set("bqtf"):
         raise ValueError
     return Services(_str=''.join(set(string)))
 
+def validate_unicode_string(string):
+    try:
+        # Python2
+        return unicode(string)
+    except NameError:
+        # Python3
+        return string
```

Comment:
Awesome! 

---
In reply to: [74458962](https://github.com/Azure/azure-cli/pull/681#discussion_r74458962) [](ancestors = 74458962)